ios/FluentUI/Navigation/SearchBar/SearchBarTokenSet.swift

```diff
@@ -49,89 +49,93 @@ public class SearchBarTokenSet: ControlTokenSet<SearchBarTokenSet.Tokens> {
             case .backgroundColor:
                 return .uiColor({
                     switch style() {
-                    case .darkContent:
+                    case .lightContent, .onCanvas:
+                        return theme.color(.background3)
+                    case .onSystemNavigationBar:
                         return theme.color(.background5)
-                    case .lightContent:
+                    case .darkContent, .onBrandNavigationBar:
```

Comment:
I think lightContent should map to the branding colors and darkContent to the system colors?